package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "reactjs_mp",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/tatsiana27/reactjs_mp.git",
+  "author": "Tatsiana_Kavaliova@epam.com",
+  "license": "MIT",
+  "scripts": {
+    "dev": "webpack-dev-server --env.env=dev",
+    "prebuild": "rimraf dist",
+    "build": "cross-env NODE_ENV=production webpack -p --env.env=prod",
+    "test": "jest",
+    "test:e2e": "./node_modules/.bin/wdio wdio.conf.js --env local"
+  },
+  "dependencies": {
+    "jest": "^22.4.3",
```

Comment:
Remove all `^`, please

FYI: The `npm` creates `package-lock.json`. But if user uses `npm` less than 5 version. The `^` effects to download up-to-date from resource. It might be problem for running application.
[using-a-package](https://docs.npmjs.com/getting-started/using-a-package.json)

Reply:
removed